libp2p/protocols/pubsub/pubsub.nim

```diff
@@ -128,7 +128,7 @@ proc send*(p: PubSub, peer: PubSubPeer, msg: RPCMsg) {.raises: [Defect].} =
 
 proc broadcast*(
   p: PubSub,
-  sendPeers: openArray[PubSubPeer],
+  sendPeers: auto, # Iteratble[PubSubPeer]
```

Comment:
What's the effect of making it auto here?

Reply:
no need to convert hashset to seq (both can be iterated over)

Reply:
nice, makes sense.